editor/src/components/canvas/commands/command-runners.ts

```diff
@@ -0,0 +1,361 @@
+import { foldEither } from '../../../core/shared/either'
+import {
+  emptyComments,
+  isJSXElement,
+  jsxAttributeValue,
+  JSXElement,
+} from '../../../core/shared/element-template'
+import {
+  getNumberPropertyFromProps,
+  jsxSimpleAttributeToValue,
+  setJSXValuesAtPaths,
+  ValueAtPath,
+} from '../../../core/shared/jsx-attributes'
+import { ElementPath, RevisionsState } from '../../../core/shared/project-file-types'
+import {
+  EditorStatePatch,
+  EditorState,
+  modifyUnderlyingForOpenFile,
+  forUnderlyingTargetFromEditorState,
+  withUnderlyingTargetFromEditorState,
+  removeElementAtPath,
+  insertElementAtPath,
+} from '../../editor/store/editor-state'
+import { Spec } from 'immutability-helper'
+import * as EP from '../../../core/shared/element-path'
+import * as PP from '../../../core/shared/property-path'
+import {
+  getProjectContentKeyPathElements,
+  ProjectContentFile,
+  ProjectContentsTree,
+  ProjectContentTreeRoot,
+} from '../../assets'
+import { drop } from '../../../core/shared/array-utils'
+import { StrategyState } from '../canvas-strategies/interaction-state'
+import {
+  applyUtopiaJSXComponentsChanges,
+  getUtopiaJSXComponentsFromSuccess,
+} from '../../../core/model/project-file-utils'
+import {
+  AdjustNumberProperty,
+  CommandFunction,
+  PathMappings,
+  ReparentElement,
+  SetProperty,
+  UpdateSelectedViews,
+} from './commands'
+
+export const runAdjustNumberProperty: CommandFunction<AdjustNumberProperty> = (
```

Comment:
I approved the strategy but now I wonder if splitting comands and command-runners is the sensible thing, or if we should split them into files per command. 
So one file per command that holds the strategy type, creator function, and the runner.
I think in hindsight the gigantic actions.ts file was a mistake that we shouldn't repeat.